iceoryx_integrationtest/iceoryx_integrationtest/test_waitset_example.py

```diff
@@ -62,12 +62,12 @@ def generate_test_description():
         sigterm_timeout='20')
 
     return launch.LaunchDescription([
+        roudi_process,
```

Comment:
please revert this. It must be possible to start the applications before RouDi and this should also be tested

Reply:
The integrationtest is starting the processes simultaneously and as we know RouDi takes some time for starting up, in the meantime the apps wait patiently if RouDi has started.
Log from test startup in waitset example:
```bash
[INFO] [launch]: Default logging verbosity is set to INFO
test_roudi_ready (test_waitset_example.TestWaitSetExample) ... [INFO] [iox-roudi-1]: process started with pid [25744]
[INFO] [iox-ex-waitset-publisher-2]: process started with pid [25746]
[INFO] [iox-ex-waitset-gateway-3]: process started with pid [25748]
[INFO] [iox-ex-waitset-grouping-4]: process started with pid [25750]
[INFO] [iox-ex-waitset-individual-5]: process started with pid [25753]
[INFO] [iox-ex-waitset-sync-6]: process started with pid [25755]
[iox-roudi-1] 2021-03-22 11:12:28.530 [ Info  ]: No config file provided. Using '/etc/iceoryx/roudi_config.toml'
[iox-roudi-1] Log level set to: [ Debug ]
[iox-roudi-1] 2021-03-22 11:12:28.530 [Warning]: IPC channel still there, doing an unlink of /roudi
[iox-ex-waitset-publisher-2] No server for unix domain socket "/tmp//roudi"
[iox-ex-waitset-publisher-2] 2021-03-22 11:12:28.530 [Warning]: RouDi not found - waiting ...
[iox-ex-waitset-gateway-3] 2021-03-22 11:12:28.530 [Warning]: IPC channel still there, doing an unlink of iox-ex-waitset-gateway
[iox-ex-waitset-gateway-3] 2021-03-22 11:12:28.530 [ Debug ]: reopen RouDi's IPC channel!
[iox-ex-waitset-gateway-3] 2021-03-22 11:12:28.530 [Warning]: RouDi not found - waiting ...
[iox-ex-waitset-grouping-4] 2021-03-22 11:12:28.531 [ Debug ]: reopen RouDi's IPC channel!
[iox-ex-waitset-grouping-4] 2021-03-22 11:12:28.531 [Warning]: RouDi not found - waiting ...
[iox-ex-waitset-individual-5] 2021-03-22 11:12:28.531 [ Debug ]: reopen RouDi's IPC channel!
[iox-ex-waitset-individual-5] 2021-03-22 11:12:28.531 [Warning]: RouDi not found - waiting ...
[iox-ex-waitset-sync-6] 2021-03-22 11:12:28.531 [ Debug ]: reopen RouDi's IPC channel!
[iox-ex-waitset-sync-6] 2021-03-22 11:12:28.531 [Warning]: RouDi not found - waiting ...
[iox-roudi-1] SharedMemory still there, doing an unlink of /iceoryx_mgmt
[iox-roudi-1] Reserving 64550400 bytes in the shared memory [/iceoryx_mgmt]
[iox-roudi-1] [ Reserving shared memory successful ] 
[iox-roudi-1] 2021-03-22 11:12:28.560 [ Debug ]: Registered memory segment 0x7eff8a551000 with size 64550392 to id 1
[iox-roudi-1] SharedMemory still there, doing an unlink of /apex
[iox-roudi-1] Reserving 149134400 bytes in the shared memory [/apex]
[iox-roudi-1] [ Reserving shared memory successful ] 
[iox-roudi-1] 2021-03-22 11:12:28.629 [ Debug ]: Roudi registered payload segment 0x7eff81717000 with size 149134400 to id 2
[iox-roudi-1] RouDi is ready for clients
ok
test_waitset_grouping (test_waitset_example.TestWaitSetExample) ... [iox-ex-waitset-publisher-2] 2021-03-22 11:12:28.730 [ Debug ]: RouDi IPC Channel found!
[iox-ex-waitset-publisher-2] 2021-03-22 11:12:28.730 [Warning]: ... RouDi found.
[iox-ex-waitset-gateway-3] 2021-03-22 11:12:28.730 [ Debug ]: RouDi IPC Channel found!
[iox-ex-waitset-gateway-3] 2021-03-22 11:12:28.730 [Warning]: ... RouDi found.
[iox-ex-waitset-grouping-4] 2021-03-22 11:12:28.731 [ Debug ]: RouDi IPC Channel found!
[iox-ex-waitset-grouping-4] 2021-03-22 11:12:28.731 [Warning]: ... RouDi found.
[iox-ex-waitset-individual-5] 2021-03-22 11:12:28.731 [ Debug ]: RouDi IPC Channel found!
[iox-ex-waitset-individual-5] 2021-03-22 11:12:28.731 [Warning]: ... RouDi found.
[iox-ex-waitset-sync-6] 2021-03-22 11:12:28.732 [ Debug ]: RouDi IPC Channel found!
[iox-ex-waitset-sync-6] 2021-03-22 11:12:28.732 [Warning]: ... RouDi found.
```